ui/package.json

```diff
@@ -3,17 +3,26 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.9.3",
+    "@emotion/styled": "^11.9.3",
     "@fortawesome/fontawesome-free": "^5.15.4",
+    "@mui/icons-material": "^5.8.4",
+    "@mui/lab": "^5.0.0-alpha.86",
```

Comment:
Kas on vaja kindlasti `alpha` releaset kasutada? Alpha üldiselt tähendab, et tegu on nö testimata `work-in-progress` arendus releasega. Kodulehelt näitab, et stabiilne 5.0.0 versioon peaks ka olemas olema.